core/lexicon/en/file.inc.php

```diff
@@ -71,8 +71,9 @@
 $_lang['file_folder_parent'] = 'Parent Directory';
 $_lang['file_folder_parent_desc'] = 'Relative to the basePath of the media source';
 $_lang['file_folder_remove'] = 'Delete Directory';
-$_lang['file_folder_remove_confirm'] = 'Are you sure you want to delete the directory: "[[+directory]]"?<br />This could potentially break your website.';
+$_lang['file_folder_remove_confirm'] = 'Are you sure you want to delete the directory: “[[+directory]]”?<br>This could potentially break your website.';
```

Comment:
I am fine using the right typography here, but the whole docs are different with double quotes.

Reply:
Yes, I know. But you've got to start somewhere. I can address the other places in a different PR. I don't see it as a problem to update them over time, as it's not a highly-visible change.